lib/maintenance_tasks.rb

```diff
@@ -89,4 +89,11 @@ module MaintenanceTasks
   #
   #   @return [Proc] generates a hash containing the metadata to be stored on the Run
   mattr_accessor :metadata, default: nil
+
+  # @!attribute task_stuck_threshold
+  #  @scope class
+  #  The threshold in seconds after which a task is considered stuck.
+  #
+  #  @return [Integer] the threshold in seconds after which a task is considered stuck.
+  mattr_accessor :task_stuck_threshold, default: 5.minutes
```

Comment:
I don't think it can be an integer since we call `#ago` on it in `#stuck?` -- I think it needs to be an `AS::Duration`.

```suggestion
  #  The duration after which a task is considered stuck and can be force cancelled.
  #
  #  @return [ActiveSupport::Duration, Numeric] duration after which a task is considered stuck.
  mattr_accessor :task_stuck_threshold, default: 5.minutes
```